### PR TITLE
increase seed size to 64 bits

### PR DESCRIPTION
**Pull request by gahtan-syarif**
fast chess supports 64 bit seeds which allows for far more unique opening sequences

### CODE REVIEW

File: worker/games.py [hunk 1]
```diff
@@ -1297,7 +1297,7 @@ def run_games(
     if "start" in task:
         print("Variable task sizes used. Opening offset = {}".format(opening_offset))
     start_game_index = opening_offset + input_total_games
-    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2**30)
+    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2**64)
```
Comment (vondele) [inline worker/games.py:1300]: are we sure the result fits a 64bit int, including the sign ?

Comment (gahtan-syarif) [reply worker/games.py:1300]: Yes, the SHA1 itself is a 160 bit value which is then modulod to 2^64

Comment (vondele) [reply worker/games.py:1300]: my point is that the max size of an 64 bit int is  2^63-1 so if we want to fit it, I don't think we can take modulo 2^64

Comment (gahtan-syarif) [reply worker/games.py:1300]: > my point is that the max size of an 64 bit int is 2^63-1 so if we want to fit it, I don't think we can take modulo 2^64

The max size of a 64 bit (u)int is 2^64-1, which you can get by modulo 2^64

Comment (gahtan-syarif) [reply worker/games.py:1300]: The result of a modulo operation is always positive

Comment (vondele) [reply worker/games.py:1300]: ok, python modulo is indeed.

**Pull request closed without merging**